etc/kernel-launchers/docker/scripts/launch_docker.py

```diff
@@ -81,21 +79,24 @@ def launch_docker_kernel(connection_file, response_addr, spark_context_init_mode
 
 if __name__ == '__main__':
     """
-        Usage: launch_docker_kernel [connection_file] [--RemoteProcessProxy.response-address <response_addr>]
+        Usage: launch_docker_kernel 
+                    [--RemoteProcessProxy.kernel-id <kernel_id>]
+                    [--RemoteProcessProxy.response-address <response_addr>]
```

Comment:
is there a case where we need the original key from the connection document?

Reply:
The connection file is not used - only the kernel-id is pulled from its name.  The launchers generate a key and send it back via the communication port.